athena/models/speech_transformer.py

```diff
@@ -259,6 +267,35 @@ def restore_from_pretrained_model(self, pretrained_model, model_type=""):
         else:
             raise ValueError("NOT SUPPORTED")
 
+    def deploy(self):
```

Comment:
We need some comments for this function, something like "frozen graph creation in tf 2.0 requires keras model, so we have to build it again"